extra_capsule_descriptions:
  arm_7_link:
     position: [0.0, 0.0, 0.1]
     orientation: [0.0, 0.0, 0.0, 1.0]
     l: 0.15
     r: 0.12
